pythainlp/tokenize/ssg.py

```diff
@@ -5,4 +5,7 @@
 
 
 def segment(text: str) -> List[str]:
+    if not text or not isinstance(text, str):
```

Comment:
This condition appears [many times][ref] in the code. I think we should consider refactor it.


[ref]: https://github.com/PyThaiNLP/pythainlp/search?q=%22if+not+text+or+not+isinstance%28text%2C+str%29%22&unscoped_q=%22if+not+text+or+not+isinstance%28text%2C+str%29%22

Reply:
@heytitle `isinstance(text, str)` only  ?

Reply:
I mean this `if`-condition appears many times.